WooCommerce new documentation (#1299)
mercadopago/devsite-docs: guides/plugins/woocommerce/testing.en.md

# Testing Purchases

> With the store in test mode, you can test a purchase before enabling it for the public. To do this, perform the entire purchase flow until completion at checkout and select the option **credit card payment**, as shown in the instructions below.

Reviewer: ```suggestion
With the store in test mode, you can test purchases before enabling them for the public. To do this, perform the entire purchase flow until completion at checkout and select the option **credit card payment**, as shown in the instructions below.
```

1. Logout of your Mercado Pago account so that you can simulate the purchase as a customer not associated with Mercado Pago.
2. Access your store's homepage on WordPress.
3. Select a product and click _Comprar > Ver carrinho_.
4. Choose any shipping option and click _Concluir compra_.
5. Fill in the required information and select the payment option with the Mercado Pago solution you have configured.

> 6. Select **credit card payment** and use the credit card information available on our page [Test local credit cards](https://www.mercadopago.com.br/developers/en/guides/resources/localization/local-cards).

Reviewer 2: ```suggestion
6. Select **credit card payment** and use the credit card information available on our page [Test local credit cards](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/localization/local-cards).
```

> 7. When you finish completing the fields, click _Finalizar compra_.

Author: ```suggestion
7. When you finish completing the fields, click on _Finalizar compra_.
```

> If the test runs without errors, it means your store is ready to sell. However, we recommend you to perform additional tests so that you can view the different payment results.

Reviewer: ```suggestion
If the test runs without errors, it means your store is ready to sell. However, we recommend you perform additional tests so that you can view the different payment results.
```
